Data_Publication_Flow.ipynb

```diff
@@ -403,6 +406,30 @@
     "print(metadata)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Viewing Search Records\n",
+    "\n",
+    "Globus Search records can also be viewed at https://petreldata.net. Portals are a good way to visually present a large number of search results for users. \n",
+    "\n",
+    "The portal shown uses the same Globus SDK calls for generating queries that you will use below. Note the facets on the left, and the filter query params in the address bar. Each of these boil down into simple parameters passed into the Globus SDK."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from urllib.parse import quote_plus\n",
+    "search_index = 'globus-tutorial'\n",
+    "subject_enc = quote_plus(quote_plus(subject))\n",
```

Comment:
I really wanted to avoid this double quoting line, but there's still a problem with the portal where it automatically unquotes URL chunks and processes them differently, resulting in an incorrect subject and causing a 404. Double-quoting prevents this, allowing the subject to be treated as a 'normal' string of characters.